back-end/seed_all_data.py

```
@@ -4,204 +4,268 @@
+                        'chatroom_id': row['chatroom_id'],
+                        'sender_id': row['sender_id'],
+                        'content': row['content'],
+                        'is_read': is_read_value,
+                        'created_at': row['created_at'],
+                    }
+                )
+            except IntegrityError:
+                print_warning(
+                    f"⚠ 메시지 ID {row['id']} 생성 중 무결성 오류. FK 확인 필요."
+                )
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)